lib/property_table.ex

```diff
@@ -85,7 +85,9 @@ defmodule PropertyTable do
     matched for triggering events. See `PropertyTable.Matcher`.
   * `:tuple_events` - set to `true` for change events to be in the old tuple
     format. This is not recommended for new code and hopefully will be removed
-    in the future.
+    in the future. Prefer `:event_transfer`
```

Comment:
```suggestion
    in the future. Prefer `:event_transformer`
```
Is this supposed to be `event_transformer`?

Reply:
Yes, thanks!